deep_qa/training/text_trainer.py

```diff
@@ -277,7 +277,7 @@ def _set_max_lengths_from_model(self):
         """
         raise NotImplementedError
 
-    def _set_max_lengths_from_model_input(self, input_slice):
+    def set_max_lengths_from_model_input(self, input_slice):
```

Comment:
I think this needs a more specific name, because it's only doing a particular kind of length.